app/questionnaire/rules.py

```diff
@@ -44,7 +44,7 @@ def evaluate_goto(goto_rule, metadata, answer_store, group_instance):
                 if len(filtered) == 1:
                     condition_met = evaluate_rule(when_rule, filtered[0]['value'])
                 elif len(filtered) > 1:
-                    logger.warning("Condition will not met: Multiple (%d) answers found" % len(filtered))
+                    logger.warning("Condition will not met: Multiple ({:d}) answers found".format(len(filtered)))
```

Comment:
I think log formats should be comma based

`logger.warning("Condition will not met: Multiple (%d) answers found", len(filtered))`

See https://docs.python.org/3.5/library/logging.html#logging.Logger.debug